specviz/plugins/model_editor/model_editor.py

```diff
@@ -368,11 +341,11 @@ def _on_fit_clicked(self, spectrum_data_item=None, eq_pop_up=True):
                 else:
                     parameter = getattr(fit_mod, param_name)
 
-                model_item.child(cidx, 1).setText(str(parameter.value))
+                model_item.child(cidx, 1).setText("{:0.5g}".format(parameter.value))
```

Comment:
Note that when the values for the fitting params in `compose_fittable_models` are derived from the text of this column, not the data. This means the rounding you do here will show up in the plot itself and the composed model. An alternative would be to derive that value from the column's `data` attribute in `compose_fittable_models` and update that attribute each time the user changes the text.

Reply:
> Note that when the values for the fitting params in compose_fittable_models are derived from the text of this column

This should absolutely not be the case, and it's probably my fault that it is. The text of a `QStandardItem` should be not ever be considered as representative of actual stored data.